api/cloudprotocol/cloudprotocol.go

```diff
@@ -359,6 +356,7 @@ type ServiceInstanceAlert struct {
 
 // AlertItem alert item structure.
 type AlertItem struct {
+	NodeID    string      `json:"nodeId"`
```

Comment:
NodeID not relevant for all alert items. We have to add nodeID to the items it is relevant for,